From a243794b97a3b39d0ff8cef63f945feebf4d6b4f Mon Sep 17 00:00:00 2001
From: "Panda,SushreeX" <sushreex.panda@intel.com>
Date: Thu, 7 Sep 2023 23:31:14 +0530
Subject: [PATCH] Fix net node permission for ADL-PS Platform to pass
 NetdSELinuxTest.

Added the missing net node contexts 0000:00:0a.0 to pass the NetdSELinuxTest#CheckProperMTULabels

Tracked-On: OAM-112139
Signed-off-by: Panda,SushreeX <sushreex.panda@intel.com>

diff --git a/ethernet/common/genfs_contexts b/ethernet/common/genfs_contexts
index 16d1e33..23c8947 100644
--- a/ethernet/common/genfs_contexts
+++ b/ethernet/common/genfs_contexts
@@ -3,6 +3,7 @@ genfscon sysfs /devices/pci0000:00/0000:00:13.2/0000:03:00.0/net u:object_r:sysf
 genfscon sysfs /devices/pci0000:00/0000:00:13.1/0000:02:00.0/net u:object_r:sysfs_net:s0
 genfscon sysfs /devices/pci0000:00/0000:00:04.0/net u:object_r:sysfs_net:s0
 genfscon sysfs /devices/pci0000:00/0000:00:07.0/net u:object_r:sysfs_net:s0
+genfscon sysfs /devices/pci0000:00/0000:00:0a.0/net u:object_r:sysfs_net:s0
 genfscon sysfs /devices/pci0000:00/0000:00:06.0/net u:object_r:sysfs_net:s0
 genfscon sysfs /devices/pci0000:00/0000:00:06.0/usb1/1-7/1-7.4/1-7.4:1.0/net u:object_r:sysfs_net:s0
 genfscon sysfs /devices/pci0000:00/0000:00:09.0/virtio2/net u:object_r:sysfs_net:s0
-- 
2.34.1

